### ISSUE TITLE
Can't run setup.py

### ISSUE THREAD

**Issue by ndr3w221** [malwaredllc/byob#96]
```

$ python setup.py                                                                                                                                                            Traceback (most recent call last):                                                                                                                                             File "setup.py", line 61, in                                                                                                                                           main()                                                                                                                                                                     File "setup.py", line 18, in main                                                                                                                                              logging.basicConfig(level=logging.DEBUG, handler=logging.StreamHandler())                                                                                                  File "C:\Program Files\Python37\lib\logging\__init__.py", line 1921, in basicConfig                                                                                            raise ValueError('Unrecognised argument(s): %s' % keys)                                                                                                                  ValueError: Unrecognised argument(s): handler
--
```
```

$ python -V                                                                                                                                                                  Python 3.7.2
--
```

**Comment by ndr3w221** [comment]
Traceback (most recent call last):                                                                                                                                             File "setup.py", line 61, in                                                                                                                                           main()                                                                                                                                                                     File "setup.py", line 18, in main                                                                                                                                              logging.basicConfig(level=logging.DEBUG, handler=logging.StreamHandler())                                                                                                  File "C:\Program Files\Python37\lib\logging\__init__.py", line 1921, in basicConfig                                                                                            raise ValueError('Unrecognised argument(s): %s' % keys)                                                                                                                  ValueError: Unrecognised argument(s): handler
--

**Comment by malwaredllc** [comment]
I haven't been able to reproduce this error. What OS are you running it on? Which version of Python are you using?

**Comment by zing145** [comment]
windows 10 pro and python 3.7

**Comment by zing145** [comment]
im having the same problem

**Comment by malwaredllc** [comment]
Try running it with python 2 really quick and let me know if there's still an error. That will help isolate the issue.

**Comment by intrct** [comment]
After reading the setup.py file, I recommend using the nested pip module within python for WIndows OS, as it is a bit cleaner;

In place of "pip install <requirement", try "python -m pip install <requirement>" -- this should be functional for 3.x, and the latest 2.7.x

**Comment by intrct** [comment]
See if the changes here do anything for you... I don't have a Windows system to test on at the moment, apologies.

https://github.com/intrct/byob/tree/issue%2396

**Comment by netzulo** [comment]
hey! im testing this at Win/lin/arm environments and workaround from @intrct working for me

at install on windows systems down virtual environment/or not, install requirements like this : ```python -m pip install -r byob\requirements.txt```

**Comment by intrct** [comment]
I think standardizing on the python -m pip install -r <requirements> is an ideal scenario @netzulo ;

This is backwards compatible with 2.x, but BYOB really needs to focus on working on 3.x and worry less about 2.x; it will be deprecated soon.

**Comment by malwaredllc** [comment]
This issue has been fixed in pull request #99
